packages/pulpcore/python-dotenv/python-dotenv.spec

```diff
@@ -44,11 +44,13 @@ sed -i 's/long_description = f.read.*/long_description = "dotenv"/' setup.py
 %files -n python3-%{srcname}
 %license LICENSE
 %doc README.md
-%{_bindir}/dotenv
```

Comment:
Same

Reply:
I just pushed a commit removing a couple of buildrequirements and requirements from this spec file. https://github.com/theforeman/foreman-packaging/pull/4886/commits/957069045e63410ce80efc530b7f27e2e34ac7d8 ..You think that's fine?

Reply:
yeah, the build/requirement drop seems fine, the sed might still be needed (sorry that my comment was off multiple lines), we'll see

Reply:
removing that `%{_bindir}/dotenv` was too much :(

Reply:
Added that and the above bit back..